usecase.go

```diff
@@ -46,6 +46,26 @@ func (c CacheHero) Get(key string) (string, error) {
 	return value, nil
 }
 
+func (c CacheHero) MGet(keys ...string) (map[string]string, error) {
+	values, err := c.client.MGet(keys...)
+	if err != nil {
+		return nil, fmt.Errorf("cachehero: %w", err)
+	}
+
+	valuesMap := make(map[string]string)
+	if len(values) == len(keys) {
```

Comment:
Si redis no encuentra alguna de las llaves, retorna un error?, si retorna un error aquí válida si es diferente, para devolver el mapa vacío y el error para sacar esta lógica del if

Reply:
Redis siempre devuelve los valores, o sea si no encuentra las llaves que se le pide este devolverá un array de nil, por esta razón dentro del for cuando quiero hacer la relación valido si es nil para darle un valor vacío.
Únicamente devolvería un error si no se le pasa al menos un key

Reply:
Cambia la lógica, si el tamaño de valores es cero, devuelve el mapa vacío y nil en el error y sacas la lógica que tienes actualmente fuera

Reply:
no, ni siquiera es necesario la validación, si es nil, simplemente no entra al for